spec/jobs/resend_validation_emails_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Jobs::ResendValidationEmails do
+  before do
+    Jobs.run_immediately!
+  end
+
+  it 'should email inactive users' do
+    user = Fabricate(:user, active: false, created_at: 3.days.ago)
+
+    expect { described_class.new.execute({}) }
+      .to change { EmailLog.count }.by(1)
+      .and change { user.email_tokens.count }.by(1)
+
+    expect(user.custom_fields['activation_reminder']).to be_truthy
```

Comment:
We try not to use `be_truthy` and `be_falsy` anymore

```suggestion
    expect(user.custom_fields['activation_reminder']).to eq(true)
```